qiita_pet/templates/study_description_templates/raw_data_editor_tab.html

```diff
@@ -60,7 +60,19 @@
           </li>
         </ol>
         <br/>
-        <a class="btn btn-primary" onclick="add_prep_template({{raw_data_id}});">Add prep template</a>
+        <table border="0">
+          <tr>
+            <td>
+              <a class="btn btn-primary" onclick="add_prep_template({{raw_data_id}});">Add prep template</a>
+            </td>
+            <td>&nbsp; &nbsp;</td>
+            <td>
+              <a class="btn btn-danger" onclick="if ({{len(available_prep_templates)}} > 0) { alert('This raw data can\'t be erased. First remove the attached prep templates') } else { delete_raw_data({{raw_data_id}}) }">
```

Comment:
What I meant is that the delete button should only be visible by the
name of the raw data when that object is selected on the interface i.e.
after you click on that tab.

On (Feb-10-15|13:00), Antonio Gonzalez wrote:
> > @@ -60,7 +60,19 @@
> >            </li>
> >          </ol>
> >          <br/>
> > -        <a class="btn btn-primary" onclick="add_prep_template({{raw_data_id}});">Add prep template</a>
> > +        <table border="0">
> > +          <tr>
> > +            <td>
> > +              <a class="btn btn-primary" onclick="add_prep_template({{raw_data_id}});">Add prep template</a>
> > +            </td>
> > +            <td>&nbsp; &nbsp;</td>
> > +            <td>
> > +              <a class="btn btn-danger" onclick="if ({{len(available_prep_templates)}} > 0) { alert('This raw data can\'t be erased. First remove the attached prep templates') } else { delete_raw_data({{raw_data_id}}) }">
> 
> Text, yes. Not sure what you mean with tab only on focus, isn't that the case already? 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/842/files#r24449989